microsoft-edge/webview2/concepts/windowed-vs-visual-hosting.md

```diff
@@ -87,30 +184,67 @@ The `CoreWebView2Controller` properties and methods:
 
 
 <!-- ====================================================================== -->
-## Windowed hosting
+## Windowed hosting and Window to Visual hosting
+
+In the Windowed hosting mode, WebView2 content is hosted directly in a window.  You can have multiple HWNDs in your app that will each be used as a WebView component to access web content.  The benefit of this is that some of the Input/Output commands are handled for you by the OS or by the framework. However, you will still need to handle some aspects of window management.
 
-Windowed hosting can be described as an HWND that stores information.  You can have multiple HWNDs in your app that will each be used as a WebView component to access web content.  Some of the Output/Input commands are handled for you by the framework you choose; however, you will still need to handle window management.
+In the Window to Visual hosting mode, content is hosted slightly differently, but has the same benefits and window-management requirements as Windowed hosting.
 
-Benefits for Windowed hosting include:
+
+<!-- ------------------------------ -->
+#### Benefits of Windowed hosting
 
 * Zooming and rasterization scales (for example, menus, context menus, and so on) will also automatically scale to the app's parent `HWND`. It also handles how the WebView manages being focused and tabbing in/out of itself when it reaches the final element.
 
 * The app handles keyboard accelerators and keyboard shortcuts when in the WebView. For example, **Ctrl+C** in a WebView will know that you are trying to copy content in the WebView, and not pressing **Ctrl** and **C** separately.
 
 * You don't have to manage the various composition-based rendering (for example, Inputs, Outputs, and Accessibility controls) if you don't want to.
 
+
+<!-- ------------------------------ -->
+#### Benefits of Window to Visual hosting
+
+* Different apps that share a WebView2 user data folder can have different DPI awarenesses.
+
+* Different apps that share a WebView2 user data folder can have different integrity levels.
+
+* Different apps that share a WebView2 user data folder will no longer be able to cause each other to hang.
+
+* Changing monitor scale when hosting a WebView2 in a VSTO add-in no longer sporadically hang the application.
+
 For general information regarding Window management and `HWND` functionality, see [About Windows](/windows/win32/winmsg/about-windows).
 
 
 <!-- ------------------------------ -->
-#### Window management
+#### APIs for Windowed hosting and Window to Visual hosting
+
+The following APIs can be used when configuring WebView2 for Windowed hosting or Window to Visual hosting.
+<!-- todo: are the below tab lists a straight copy of "Overview of APIs"?  link to there instead?
```

Comment:
are the below tab lists a straight copy of "Overview of APIs"? link to there instead?

Present article:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/windowed-vs-visual-hosting?branch=pr-en-us-3271
Overview of APIs:

Window management - no equiv section?
   [Sizing, positioning, and visibility](https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3271#sizing-positioning-and-visibility)
   [Zooming](https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3271#zooming)
[Rasterization scale](https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3271#rasterization-scale)
[Focus and tabbing](https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3271#focus-and-tabbing)
[Parent window](https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3271#parent-window)
[Keyboard accelerators](https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3271#keyboard-accelerators)
[Default background color](https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3271#default-background-color)

Reply:
in every API section in article, linked to same/similar/equivalent section in "Overview of APIs" article.  for maintainability; we can easily compare the two sections to see if same, and why heading sometimes differs.